jenkins_jobs/upload_build_artifacts/script.sh

```diff
@@ -24,7 +24,7 @@ mkdir $BUILD_TIMESTAMP
 rsync_upload --recursive ./$BUILD_TIMESTAMP
 
 rsync_upload STATUS
-rsync_upload --recursive repository
+rsync_upload --recursive --copy-links repository
```

Comment:
This is not necessary. When you used `--copy-links` above to archive the files from slave to master, rsync already transformed all links to files/directories.